### PR TITLE
resolved: improve debug logging on incoming messages a bit

### PR DESCRIPTION
**Pull request by poettering**
This is the last PR split out of #17535.

### CODE REVIEW

**Comment by yuwata** [comment]
LGTM.

File: src/resolve/resolved-manager.c [hunk 1]
```diff
@@ -926,8 +926,10 @@ int manager_recv(Manager *m, int fd, DnsProtocol protocol, DnsPacket **ret) {
                         p->ifindex = manager_find_ifindex(m, p->family, &p->destination);
         }
 
-        *ret = TAKE_PTR(p);
+        log_debug("Received %s UDP of size %zu, ifindex=%i, ttl=%i, fragsize=%zu",
```
Comment (yuwata) [inline src/resolve/resolved-manager.c]: `UDP` -> `message` or `packet`??

"Received dns UDP of size..." sounds strange for me.

**Comment by poettering** [comment]
slightly updated the message, as suggested. ptal.

**Comment by yuwata** [comment]
Thanks. LGTM.

**Comment by keszybz** [comment]
Let's merge this without waiting for all CI. We don't test debug logs, so if it compiles, additional testing is not going to tell us anything.

**Pull request merged**